fixcloudutils/redis/event_stream.py

```diff
@@ -165,8 +171,13 @@ async def _listen(self) -> None:
                     self.group, self.listener, {self.stream: self.__readpos}, count=self.batch_size, block=1000
                 )
                 self.__readpos = ">"
-
-                await self._handle_stream_messages(messages)
+                if self.parallelism:
+                    await self._handle_stream_messages_parallel(messages, self.parallelism)
+                else:
+                    await self._handle_stream_messages(messages)
+            except RuntimeError as r:
+                if len(r.args) > 0 and r.args[0] == "no running event loop":
+                    raise r
```

Comment:
This catches all other runtime errors and probably differs from what you wanted.